Services/Container/classes/class.ilContainerImporter.php

```diff
@@ -134,7 +134,8 @@ protected function handleOfflineStatus(string $xml, ilImportMapping $mapping)
                 if ($this->isRootNode($obj->getRefId(), $mapping)) {
                     $obj->setOfflineStatus(true);
                 } else {
-                    $obj->setOfflineStatus(false);
+                    // take the offline state of the XML file, and map it back: online to online and offline to offline
+                    $obj->setOfflineStatus(!((string)$offline === "0"));
```

Comment:
This violates our coding standard, a space is missing after the `(string)` cast.